README.md

```diff
@@ -140,6 +140,7 @@ project_id1          GCPAppID     1071284184432
 | kpt                                                  |    x    |         |       |            x            |            |
 | kubectl                                              |    x    |         |       |            x            |            |
 | local-extract                                        |    x    |         |       |            x            |            |
+| Docker in Docker (dind)                              |         |    x    |       |                         |            |
```

Comment:
We should not add "Docker in Docker (dind)" here, as it is not a gcloud component. (See `gcloud components list` for a list of all of the components.)

Reply:
Should it be documented somewhere else then? I think it’s worth documenting, since the `docker` executable is available in all images, but dind only works in the updated alpine image.

Reply:
I would recommend adding a new section above "### Legacy image" called "### Docker-in-docker support", and add a few sentences there.